src/index.html

```diff
@@ -1,17 +1,170 @@
 <!DOCTYPE html>
-<html lang="en">
-
+<html lang="pl">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="title" content="Patrycja Kołaczyńska - architektura" />
+    <meta name="description" content="Patrycja Kołaczyńska - architektura" />
+    <title>Patrycja Kołaczyńska - architektura</title>
+    <script type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule="" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.js"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500&family=Rozha+One&display=swap" rel="stylesheet">
```

Comment:
tutaj dodaj `rel="preconnect"` i daj nad script. Pokazujesz przeglądarce, że to jest coś ważnego do wyrenderowania w pierwszej kolejności. Do czego potrzebujesz script? Może możesz je umieścić na dole? To lepsze dla performancu :)